cmd/Makefile.am

```diff
@@ -386,23 +389,33 @@ endif
 snap-confine/snap-confine.apparmor: snap-confine/snap-confine.apparmor.in Makefile
 	sed -e 's,[@]LIBEXECDIR[@],$(libexecdir),g' <$< >$@
 
+CLEANFILES += snap-confine/snap-confine.apparmor
+
+if APPARMOR
+install-data-local: snap-confine/snap-confine.apparmor
+endif
+
+install-data-local:
+if APPARMOR
 # Install the apparmor profile
 #
 # NOTE: the funky make functions here just convert /foo/bar/froz into
 # foo.bar.froz The inner subst replaces slashes with dots and the outer
 # patsubst strips the leading dot
-install-data-local:: snap-confine/snap-confine.apparmor
-if APPARMOR
-	install -d -m 755 $(DESTDIR)/etc/apparmor.d/
-	install -m 644 snap-confine/snap-confine.apparmor $(DESTDIR)/etc/apparmor.d/$(patsubst .%,%,$(subst /,.,$(libexecdir))).snap-confine
+	install -d -m 755 $(DESTDIR)$(APPARMOR_SYSCONFIG)/
+	install -m 644 snap-confine/snap-confine.apparmor $(DESTDIR)$(APPARMOR_SYSCONFIG)/$(patsubst .%,%,$(subst /,.,$(libexecdir))).snap-confine
 endif
-	install -d -m 755 $(DESTDIR)/var/lib/snapd/apparmor/snap-confine/
-
+	install -d -m 755 $(DESTDIR)$(snapdstatedir)/apparmor/snap-confine/
 # NOTE: The 'void' directory *has to* be chmod 111
-install-data-local::
-	install -d -m 111 $(DESTDIR)/var/lib/snapd/void
+	install -d -m 111 $(DESTDIR)$(snapdstatedir)/void
+
+uninstall-local:
+if APPARMOR
+	rm -f $(DESTDIR)$(APPARMOR_SYSCONFIG)/$(patsubst .%,%,$(subst /,.,$(libexecdir))).snap-confine
+endif
+	rmdir $(DESTDIR)$(snapdstatedir)/void || true
 
-install-exec-hook::
+install-exec-hook:
```

Comment:
This was deliberate. This way the hook has several separate rules and they all execute. Why do you want to change it to a normal rule?

Reply:
There is only one rule named like that. There is no need for `::`

Reply:
Ah apologies, the diff confused me